drivers/dai/intel/ssp/ssp.h

```diff
@@ -15,12 +15,6 @@
 #define DAI_INTEL_SSP_MASK(b_hi, b_lo)	\
```

Comment:
can you replace this with GENMASK?

Reply:
yep, there are more cleanup to do, just sent these ones first